### PR TITLE
Add Wagmi icon

### LINKED ISSUE #10817: Request: Wagmi
### Brand Name

wagmi

### Website

https://wagmi.sh/

### Popularity Metric

https://similarweb.com/website/wagmi.sh
Rank: #373172

### Official Resources for Icon and Color

svg: https://raw.githubusercontent.com/wevm/wagmi/main/.github/logo-light.svg
color: #FFFFFF

### Additional Comments

there is another version of the logo with just a different color:
https://raw.githubusercontent.com/wevm/wagmi/main/.github/logo-dark.svg

### PR DESCRIPTION
**Pull request by anhthang**
image

**Issue:** closes #10817  

**Popularity metric:**
Rank 425,059
5.7k ⭐ on Github

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

SVG extracted from the webstie

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
There is a more icon-appropriate source in [this file](https://wagmi.sh/favicon.svg) which we should base our logo on. Don't need to update the source, as it's in the source code of the homepage!

File: icons/wagmi.svg
Comment (adamrusted) [inline icons/wagmi.svg:1]: Seeing a _slight_ difference between PR (black) and source (white) - probably down to precision when optimizing. Could you try with an additional point of precision, @anhthang?
image

Comment (anhthang) [reply icons/wagmi.svg:1]: Increased the precision, looks good from end, please review again

**Review by adamrusted — APPROVED** [review]
Perfect, thanks @anhthang! 🎉

**Pull request merged**